projects/plugins/wpcomsh/wpcom-features/class-wpcom-features.php

```diff
@@ -704,6 +705,10 @@ class WPCOM_Features {
 			self::WPCOM_BUSINESS_AND_HIGHER_PLANS,
 			self::WPCOM_PRO_PLANS,
 		),
+		self::FIELD_FILE                        => array(
+			self::WPCOM_PERSONAL_AND_HIGHER_PLANS,
```

Comment:
I think we should exclude the WOO trial from this. 
This way we don't worry about folks that are not paying us and us storing info for free for them.